aten/src/ATen/core/dispatch/OpSchemaRegistration.h

```diff
@@ -2,17 +2,38 @@
 
 #include <ATen/core/dispatch/Dispatcher.h>
 
-// TODO Better error message when this definition is missing
+namespace c10 {
+namespace detail {
+class OpSchemaRegistrar final {
```

Comment:
Docs?

Reply:
Well, FunctionSchema is the OpSchema now. No docs because private.